orttraining/orttraining/test/python/checkpoint/orttraining_test_checkpoint_aggregation.py

```diff
@@ -88,12 +99,115 @@ def test_aggregation_from_distributed_zero_mixed_precision_lamb(device='cuda', c
     test_zero_aggregation(checkpoint_dir, loaded_state_dict, is_mixedprecision=True)
 
 
+def test_aggregation_from_distributed_megatron_full_precision_adam(device='cuda', checkpoint_dir='checkpoint_dir/distributed_megatron/full_precision/adam/'):
```

Comment:
```suggestion
def test_aggregation_from_distributed_megatron_full_precision_adam(device='cuda', checkpoint_dir='checkpoint_dir/distributed_megatron/full_precision/adam/'):
```

Ideally we shouldn't use had-coded paths using forward slashes as it is no cross-platform